rules/xcodeproj.bzl

```diff
@@ -132,6 +134,7 @@ def _xcodeproj_aspect_impl(target, ctx):
             env_vars = env_vars,
             hmap_paths = depset([], transitive = _get_attr_values_for_name(deps, _SrcsInfo, "hmap_paths")),
             commandline_args = commandline_args,
+            tags = tags,
```

Comment:
instead of propogating the tags, could we skip adding a `direct` entry into the depset when `xcodeproj-ignore-as-target` is in the tags? that theoretically should let us short-circuit some work